src/sentry_webhook/index.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as sentry from '@sentry/types';
+import * as express from 'express';
+
+import {postSentryEventToSalesforce} from './post_sentry_event_to_salesforce';
+
+exports.postSentryEventToSalesforce = (req: express.Request, res: express.Response) => {
+  if (req.method !== 'POST') {
+    res.status(405).send('Method not allowed');
+    return;
+  }
+  if (!req.body) {
+    res.status(400).send('Missing request body');
+    return;
+  }
+
+  const sentryEvent: sentry.SentryEvent = req.body.event;
+  if (!sentryEvent) {
+    res.status(400).send('Missing Sentry event');
+    return;
+  }
+  // Use the request message if SentryEvent.message is unpopulated.
+  sentryEvent.message = sentryEvent.message || req.body.message;
+  postSentryEventToSalesforce(sentryEvent, req.body.project)
+      .then(() => {
+        res.status(200).send();
+      })
+      .catch((e) => {
+        console.error(e);
+        // Send an OK response to Sentry - they don't need to know about errors with posting to
```

Comment:
You'll hate me for asking but did you test if Sentry will retry a webhook if it receives, e.g, a 500?

Reply:
I did, it does not retry. Yet, I believe we shouldn't be sending them errors unless there is a problem with the request itself.

Reply:
If they don't retry, I agree.